src/Airmap/AirMapWeatherInfoManager.cc

```diff
@@ -10,9 +10,8 @@
 #include "AirMapWeatherInfoManager.h"
 #include "AirMapManager.h"
 
-
-#define WEATHER_UPDATE_DISTANCE 50000                   //-- 50km threshold for weather updates
-#define WEATHER_UPDATE_TIME     30 * 60 * 60 * 1000     //-- 30 minutes threshold for weather updates
+#define WEATHER_UPDATE_DISTANCE     50000           //-- 50km threshold for weather updates
+#define WEATHER_UPDATE_TIME_MSECS   30 * 60 * 1000  //-- 30 minutes threshold for weather updates
```

Comment:
As far as I can tell this value was incorrect. It is supposed to be milliseconds.